astroquery/mast/core.py

```diff
@@ -40,10 +44,22 @@ class ResolverError(Exception):
     pass
 
 
+class SessionTokenError(Exception):
```

Comment:
I feel like such exceptions should be in some higher level module for all sub-modules to share. Just a thought. I'll let the maintainers decide.

Reply:
I could use the more generic LoginError.

Reply:
yes, ``LoginError`` seems much better to me, thus sharing the behaviour with other modules that do authentication.

Reply:
fixed